static_src/components/users_invite.jsx

```diff
@@ -65,55 +58,60 @@ export default class UsersInvite extends React.Component {
   }
 
   get errorMessage() {
-    const err = this.props.error;
-    if (!err) return undefined;
+    const { err } = this.props;
+
+    if (!err) return;
+
     const message = err.contextualMessage;
+
     if (err.message) {
       return `${message}: ${err.message}.`;
     }
+
     return message;
   }
 
+  get entityType() {
+    return /space/.test(this.props.inviteEntityType) ? 'space' : 'organization';
```

Comment:
I think, in a future PR, we will want to have some kind of service class or utility code that takes the raw name of the entity and gives us the human-friendly version.